selecta

```diff
@@ -276,6 +266,15 @@ class Search
     end
   end
 
+  def move(direction)
```

Comment:
I think that `direction` should be inverted, so that up is `move(-1)` and down is `move(1)`. That corresponds to the movement through the array. AFAIK, there's no standard for whether "up" in a terminal is positive or negative, so there's no expectation that will be broken.